assets/base.css

```diff
@@ -354,73 +354,73 @@ h5,
   font-family: var(--font-heading-family);
   font-style: var(--font-heading-style);
   font-weight: var(--font-heading-weight);
-  letter-spacing: 0.06rem;
+  letter-spacing: calc(var(--font-heading-scale) * 0.06rem);
   color: rgb(var(--color-foreground));
   line-height: 1.3;
 }
 
 .h0 {
-  font-size: 4rem;
+  font-size: calc(var(--font-heading-scale) * 4rem);
 }
 
 @media only screen and (min-width: 750px) {
   .h0 {
-    font-size: 5.2rem;
+    font-size: calc(var(--font-heading-scale) * 5.2rem);
   }
 }
 
 h1,
 .h1 {
-  font-size: 3rem;
+  font-size: calc(var(--font-heading-scale) * 3rem);
 }
 
 @media only screen and (min-width: 750px) {
   h1,
   .h1 {
-    font-size: 4rem;
+    font-size: calc(var(--font-heading-scale) * 4rem);
   }
 }
 
 h2,
 .h2 {
-  font-size: 2rem;
+  font-size: calc(var(--font-heading-scale) * 2rem);
 }
 
 @media only screen and (min-width: 750px) {
   h2,
   .h2 {
-    font-size: 2.4rem;
+    font-size: calc(var(--font-heading-scale) * 2.4rem);
   }
 }
```

Comment:
The vendor sizing (wrapped in a `p`) isnt updating its size: https://screenshot.click/28-15-mb2ni-0t7jo.png

Code: https://screenshot.click/28-16-99w0a-ejdjh.png

Reply:
This would be considered "body text" and should be scaling with body_scale setting